src/test/java/core/basesyntax/ValidatorTests.java

```diff
@@ -0,0 +1,55 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import dao.Reader;
+import dao.ReaderImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import validator.Validator;
+import validator.ValidatorImpl;
+
+public class ValidatorTests {
+    private static Reader reader;
+    private static Validator validator;
+
+    private static List<String> validData;
```

Comment:
why all data are static? 
Before annotation doesn't require fields to be static